src/Core/src/Platform/Android/MauiTimePicker.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Android.Content;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using static Android.Views.View;
+
+namespace Microsoft.Maui
+{
+	public class MauiTimePicker : EditText, IOnClickListener
```

Comment:
Is there some overlap with the Picker? I see some derived types here: https://github.com/dotnet/maui/pull/433/files#diff-df34a6df84bad7762b1ed8a016db8d80eeb0e1c94f25562446e52424f94a196a

Also, there is a PickerManager that might have some things that we may want/need? https://github.com/dotnet/maui/pull/433/files#diff-ba5788b29220792f8b9f67a65077c6b25781a9cfd8b59ef06896671af9b24efe

Should these both be using a shared base type that is a PickerEditText and then each type of picker just ahs a thin wrapper for it?